extern/sector-storage/resources.go

```diff
@@ -268,7 +316,7 @@ var ResourceTable = map[sealtasks.TaskType]map[abi.RegisteredSealProof]Resources
 			MinMemory: 1 << 20,
 
 			MaxParallelism: 0,
-			CanGPU:         false,
+			GPUUtilization: 1.0,
```

Comment:
This should be 0 / not present (and a few below as well)

Reply:
Got overly aggressive in replacing lines with `CanGPU`. Fixed.